.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -21,13 +21,16 @@ jobs:
           - kernel
           - kernel-dragonball-experimental
           - kernel-tdx-experimental
+          - kernel-sev-experimental
```

Comment:
Is SEV support already merged in an upstream kernel?
If so, it doesn't have to be an "-experimental" kernel.

Reply:
ah yeah I will switch this back to non-experimental